platform_it/pom.xml

```diff
@@ -1180,6 +1184,14 @@
                 <extraDependency2>splice_ee</extraDependency2>
                 <extraDependency3>splice_ee</extraDependency3>
             </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.apache.spark</groupId>
+                    <artifactId>spark-avro_${scala.binary.version}</artifactId>
+                    <version>${spark.version}</version>
+                    <scope>runtime</scope>
```

Comment:
The main pom file has a slightly different dependency for HDP, pulling this artifact from databricks.  Will this conflict?  Should we make these match?  Or should we add this dependency in the main pom file only, but under each profile id in case different platforms need a different dependency?

> 
>            <id>hdp3.1.0</id>
>             <dependencies>
>                 <dependency>
>                     <!-- com.databricks.spark.avro -->
>                     <groupId>com.databricks</groupId>
>                     <artifactId>spark-avro_${scala.binary.version}</artifactId>
>                     <version>4.0.0</version>
>                     <scope>runtime</scope>
>                 </dependency>
>             </dependencies>

Reply:
spark-avro for dbaas3 profile is moved to hbase_sql, where it's configured already for other profiles